src/models/chan.js

```diff
@@ -23,18 +23,21 @@ function Chan(attr) {
 	}, attr));
 }
 
-Chan.prototype.sortUsers = function() {
-	this.users = _.sortBy(
-		this.users,
-		function(u) { return u.name.toLowerCase(); }
-	);
-
-	["+", "%", "@", "&", "~"].forEach(function(mode) {
-		this.users = _.remove(
-			this.users,
-			function(u) { return u.mode === mode; }
-		).concat(this.users);
-	}, this);
+Chan.prototype.sortUsers = function(irc) {
+	var userModeSortPriority = {};
+	irc.network.options.PREFIX.forEach(function(prefix, index) {
```

Comment:
I think it would be nicer to only pass the prefixes in the `sortUsers` function. To assist with that, we can have `Network.prototype.prefixes = function()` in `network.js` that returns the array of prefixes for that network. We can then call things like `chan.sortUsers(network.prefixes());` in a simple way (simplify tests too).

I was trying to provide it, but I noticed there is `options.PREFIX`, `serverOptions.PREFIX` and `prefixLookup`, all related. It seems to me that `sortUsers` should be using `serverOptions.PREFIX` instead of `options.PREFIX` but I didn't look at the code too closely. It would be better to keep a single source of truth and use methods to produce the others.

Also, in places calling `sortUsers()`, There are `irc.network` and `network`. Do they carry the same values?

Happy to help on this.